### ISSUE TITLE
Reconfiguring device causes device instability

### ISSUE THREAD

**Issue by webwolf3000** [rospogrigio/localtuya#1162]
After configuring tuya local, pressing configure and submit again causes the device to become unavailable.

Must completely remove the integration from tuya app and home assistant and start again from scratch.

**Comment by techtim89** [comment]
Yes!  Me too..  This is BEYOND frustrating!  I have reset my instance up three times now for this to happen and nothing I do can remove the said device which forces me to remove the entire integration.

**Comment by ThomasCr** [comment]
Had the same problem - the device was there - but no entitie.
Could fix it after redefines it more times...
